<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDateTimeCalender</class>
 <widget class="QWidget" name="TestDateTimeCalender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>265</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TestDateTimeCalender</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_01DateAndTime">
      <attribute name="title">
       <string>Date And Time</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_01Read">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>ReadCurrentDateTime</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_01Display">
           <property name="text">
            <string>String display</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="2">
          <widget class="QLineEdit" name="lineEdit_03DateTime">
           <property name="inputMask">
            <string>99.99.9999 99:99:99.999</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_04DateTime">
           <property name="text">
            <string>DateTime</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDateTimeEdit" name="dateTimeEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="displayFormat">
            <string>ddd,MMM d.M.yyyy H:m:s.z</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="pushButton_05SetDate">
           <property name="text">
            <string>SetDateFromString</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButton_04Date">
           <property name="text">
            <string>qDebug()--Date</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QTimeEdit" name="timeEdit">
           <property name="displayFormat">
            <string>HH:mm:ss.zzz A</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_03Date">
           <property name="text">
            <string>Date</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_03SetTime">
           <property name="text">
            <string>SetTimeFromString</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="lineEdit_02Date">
           <property name="inputMask">
            <string>9999-99-99</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_02Time">
           <property name="text">
            <string>Time</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDateEdit" name="dateEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="displayFormat">
            <string>dddd,MMMM yyyy/MM/dd</string>
           </property>
           <property name="date">
            <date>
             <year>2000</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEdit_01Time">
           <property name="inputMask">
            <string>99:99:99.999</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_02Time">
           <property name="text">
            <string>qDebug()--Time</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="pushButton_06DateTime">
           <property name="text">
            <string>qDebug()--DateTime</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QPushButton" name="pushButton_07SetDT">
           <property name="text">
            <string>SetDateTimeFromString</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_02Calendar">
      <attribute name="title">
       <string>Calendar</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Choosing Date:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCalendarWidget" name="calendarWidget"/>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="dateEdit_2Calendar">
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="displayFormat">
          <string>dddd,MMMM dd.MM.yyyy (ddd,MMM d/M/yy)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>calendarWidget</sender>
   <signal>clicked(QDate)</signal>
   <receiver>dateEdit_2Calendar</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
